### PR TITLE
fix(config): Add missing capScreenshareBitrate to config.js

### PR DESCRIPTION
**Pull request by jallamsetty1**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -46,14 +46,20 @@ var config = {
...
 
         // Disables the auto-play behavior of *all* newly created video element.
         // This is useful when the client runs on a host with limited resources.
         // noAutoPlayVideo: false
+
+        // Enable / disable 500 Kbps bitrate cap on desktop tracks. When enabled,
+        // simulcast is turned off for the desktop share. If presenter is turned
+        // on while screensharing is in progress, the max bitrate is automatically
+        // adjusted to 2.5 Mbps.
+        capScreenshareBitrate: 1 // 0 to disable
```
Comment (saghul) [inline config.js:62]: Please use a Boolean instead of an integer and leave it commented out, with the default value set.

Comment (saghul) [reply config.js:62]: Also, is is whitelisted already?

Comment (saghul) [reply config.js:62]: If the code still uses it as a probability we should document it IMHO

Comment (jallamsetty1) [reply config.js]: ok sure, will do that. Also its under testing which is whitelisted, we don't need to whitelist this separately right ?

Comment (saghul) [reply config.js]: Correct!

**Pull request merged**